data/json/items/comestibles/soup.json

```diff
@@ -2,7 +2,46 @@
   {
     "type": "COMESTIBLE",
     "id": "broth",
-    "name": "broth",
+    "name": { "str": "broth" },
+    "conditional_names": [
+      { "type": "COMPONENT_ID", "condition": "mushroom", "name": { "str": "mushroom %s" } },
+      { "type": "COMPONENT_ID", "condition": "morel", "name": { "str": "morel %s" } },
+      { "type": "COMPONENT_ID", "condition": "egg", "name": { "str": "egg %s" } },
+      {
+        "type": "COMPONENT_ID",
+        "condition": "dandelion",
+        "name": { "str": "dandelion %s", "str_pl": "dandelion %s" }
```

Comment:
Why does this have a plural string, but the strings for eggs and morel and mushroom above don't?

Reply:
Problem from using techniques to speed up the problem. Manually added the first three but then pasted from another list for the rest.